hw3/01-characters.js

```diff
@@ -0,0 +1,59 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+const throneSection = document.querySelector('section');
+throneSection.style.display = 'flex';
+throneSection.style.flexWrap = 'wrap';
+throneSection.style.justifyContent = 'center';
+
+function styleThroneDiv() {
+  const divToStyle = document.createElement('div');
+  divToStyle.className = 'gameOfThrones m-2 p-2 justify-content-center';
+  divToStyle.onmousemove = () => {
+    this.style.backgroundColor = '#13415b';
+    this.style.color = 'white';
+  };
+  divToStyle.onmouseout = () => {
+    this.style.backgroundColor = '';
+    this.style.color = 'black';
+  };
```

Comment:
Good catch! When I refactored this into it's own function, it caused a bug. I completely agree that CSS would be better, but since we weren't provided with a CSS file, I'm going to keep in js.